odoo/tools/mail.py

```diff
@@ -438,6 +441,7 @@ def append_content_to_html(html, content, plaintext=True, preserve=False, contai
 mail_header_msgid_re = re.compile('<[^<>]+>')
 
 email_addr_escapes_re = re.compile(r'[\\"]')
+email_localpart_unsafe_chars_re = re.compile(r'[^a-zA-Z0-9_\.\-\+]')
```

Comment:
"unsafe_chars" brings a security-related connotation, how about `unsupported_chars` or something more anchored in the RFC spec?